.github/pull_request_template.md

```diff
@@ -33,6 +33,17 @@
 # Change characteristics
 - Is this a breaking change (a change in existing functionality)? YES/NO
 - Does this change require a documentation update? YES/NO
+- Does this change require an update to any of the following submodules? YES/NO
+  - [ ] UFS-weather-model
+  - [ ] UFS-UTILS
+  - [ ] GSI
+  - [ ] GSI-utils
+  - [ ] GSI-monitor
+  - [ ] GDAS
+  - [ ] EMC verif-global
+  - [ ] wxflow
+  - [ ] GFS-utils
```

Comment:
Can we alphabetize these? Also a note to link the related PRs

Reply:
Done.  Please re-review.